<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PortScannerWindow</class>
 <widget class="QMainWindow" name="PortScannerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Port Scanner</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="ipLineEdit">
      <property name="placeholderText">
       <string>Enter IP Address</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="startPortLineEdit">
      <property name="placeholderText">
       <string>Enter Start Port</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="endPortLineEdit">
      <property name="placeholderText">
       <string>Enter End Port</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="scanTypeComboBox">
      <item>
       <property name="text">
        <string>Quick Scan (Common Ports)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Full Scan (All Ports)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>TCP Scan</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>UDP Scan</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="startButton">
      <property name="text">
       <string>Start Scan</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar"/>
    </item>
    <item>
     <widget class="QTextEdit" name="resultTextEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
